docs/configuration.md

```diff
@@ -2013,7 +2013,62 @@ Apart from these, the following properties are also available, and may be useful
     </tr>
 </table>
 
+### HTTP Security Headers
 
+Apache Spark can be configured to include HTTP Headers which aids in preventing Cross 
+Site Scripting (XSS), Cross-Frame Scripting (XFS), MIME-Sniffing and also enforces HTTP 
+Strict Transport Security.
+
+<table class="table">
+    <tr><th>Property Name</th><th>Default</th><th>Meaning</th></tr>
+    <tr>
+        <td><code>spark.ui.xXssProtection</code></td>
+        <td>None</td>
+        <td>
+            Value for HTTP X-XSS-Protection response header. You can choose appropriate value 
+            from below:
+            <ul>
+                <li>  0 (Disables XSS filtering)
```

Comment:
Markdown won't work here? I think it won't. Close the `<li>` tags though. It's not clear below what the values the users sets though. The values are `0`, `1`, or `1; mode=block` and that should be clear.

Reply:
Removed the lists, however it is being used in many places in existing configuration.md